packages/site-client/package.json

```diff
@@ -33,7 +33,7 @@
       "clean": "if-file-deleted"
     },
     "build:prod": {
-      "command": "esbuild --format=esm --target=es2022 --color --outdir=bundled --bundle --splitting --minify src/entrypoints/*.ts",
+      "command": "esbuild --color --outdir=bundled --bundle --splitting --minify src/entrypoints/*.ts",
```

Comment:
Format is picked automatically when bundling is enabled, but I couldn't figure out from the docs which format it's supposed to be: https://esbuild.github.io/api/#format

Here and above, why not keep the explicit `--format=esm` flag?

Reply:
If you specify these flags then esbuild doesn't read the tsconfig and it sets `useDefineForClassFields` to `true`, which breaks our elements.